Library/Homebrew/cask/lib/hbc/exceptions.rb

```diff
@@ -29,13 +29,30 @@ def to_s
 
   class CaskAlreadyInstalledError < AbstractCaskErrorWithToken
     def to_s
-      %Q{A Cask for #{token} is already installed. Add the "--force" option to force re-install.}
+      s = <<-EOS.undent
+        A Cask for #{token} is already installed.
+      EOS
+
+      s.concat("\n").concat(reinstall_message)
+    end
+
+    private
+
+    def reinstall_message
+      <<-EOS.undent
+        To re-install #{token}, run:
+          brew cask uninstall --force #{token}; brew cask install #{token}
```

Comment:
Should this be `&&` instead of `;` or is it unimportant if the first command somehow fails?